src/pages/send/confirm/confirm.ts

```diff
@@ -294,31 +294,30 @@ export class ConfirmPage extends WalletTabsChild {
     }
   }
 
-  private setWalletSelector(
-    coin: string,
-    network: string,
-    minAmount: number
-  ): Promise<any> {
+  private setWalletSelector(coin: string, network: string): Promise<any> {
     const parentWallet = this.getParentWallet();
     if (
       parentWallet &&
       (parentWallet.network == network && parentWallet.coin == coin)
     ) {
       return Promise.resolve();
     }
-    return new Promise(resolve => {
-      // no min amount? (sendMax) => look for no empty wallets
-      minAmount = minAmount ? minAmount : 1;
-
-      this.wallets = this.profileProvider.getWallets({
-        onlyComplete: true,
-        hasFunds: true,
-        network,
-        coin
-      });
 
-      return resolve();
+    this.wallets = this.profileProvider.getWallets({
+      onlyComplete: true,
+      hasFunds: true,
+      network,
+      coin
     });
+
+    if (_.isEmpty(this.wallets)) {
+      const msg = this.translate.instant(
+        'You are trying to send more funds than you have available. Make sure you do not have funds locked by pending transaction proposals.'
+      );
+      const title = this.translate.instant('No wallets available');
+      return Promise.reject({ msg, title });
```

Comment:
the only time you will ever reach this condition is when the wallet balance is `0` due to the flag:

`hasFunds: true`

You can have a transaction with amount 10000 ETH with only 1 ETH wallet balance, but `this.wallets` will not be empty.